ubuntudesign/django-json-redirects
# Genericise and split out YAML parsing

Fixes https://github.com/canonical-webteam/yaml-redirects/issues/6

- Split out the generic part of this module into https://github.com/canonical-webteam/views-from-yaml
- Rename this module to `canonicalwebteam.yaml-redirects`, to follow the modern pattern
- Add a `canonicalwebteam` namespace
- Rename the main function to `create_views`, inline with the `views-from-yaml` module

QA
--

Run [this PR on ubuntu.com](https://github.com/canonical-websites/www.ubuntu.com/pull/2102). Check redirects and the rest of the site still work as they did before.

Also read the code in the new module https://github.com/canonical-webteam/views-from-yaml and check it makes sense
